File: backend/src/main/java/com/tripfriend/domain/member/member/dto/LoginRequestDto.java
package com.tripfriend.domain.member.member.dto;

import jakarta.validation.constraints.NotBlank;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@AllArgsConstructor
@NoArgsConstructor
public class LoginRequestDto {

    @NotBlank(message = "아이디는 필수 입력값입니다")
    private String username;

    @NotBlank(message = "비밀번호는 필수 입력값입니다")
    private String password;
}

File: backend/src/main/java/com/tripfriend/domain/member/member/dto/LoginRequestDto.kt
package com.tripfriend.domain.member.member.dto

import jakarta.validation.constraints.NotBlank

data class LoginRequestDto(
    @field:NotBlank(message = "아이디는 필수 입력값입니다")
    val username: String = "",

    @field:NotBlank(message = "비밀번호는 필수 입력값입니다")
    val password: String = ""
)
